home/javascript/react/components/WikiBlogPostList.js

```diff
@@ -56,7 +56,7 @@ class WikiBlogPostList extends Component {
                 <LoadingSpinner loading={loading} />
                 <div className='wiki-list-controls'>
                     <span>{ next && <a href='#' onClick={this.loadNext}>Load More</a> }</span>
-                    <span>{ showAll && <a href={`https://${process.env.WIKI_HOST}${showAll}`}>See All <i className='fas fa-chevron-right' /></a>}</span>
+                    <span>{ showAll && <a className='no-icon' href={`https://${process.env.WIKI_HOST}${showAll}`}>See All <i className='fas fa-chevron-right' /></a>}</span>
```

Comment:
Do we have more than one wiki_host variable? I think we probably can just hard-code the wiki host to the one we have.

Reply:
Done